pkg/policy/validate.go

```diff
@@ -1097,8 +1097,10 @@ func checkClusterResourceInMatchAndExclude(rule kyvernov1.Rule, clusterResources
 							if rule.Generation.Namespace != policyNamespace {
 								return fmt.Errorf("path: spec.rules[%v]: a namespaced policy cannot generate resources in other namespaces, expected: %v, received: %v", rule.Name, policyNamespace, rule.Generation.Namespace)
 							}
-							if rule.Generation.Clone.Namespace != policyNamespace {
-								return fmt.Errorf("path: spec.rules[%v]: a namespaced policy cannot clone resource in other namespace, expected: %v, received: %v", rule.Name, policyNamespace, rule.Generation.Clone.Namespace)
+							if rule.Generation.Clone.Name != "" {
+								if rule.Generation.Clone.Namespace != policyNamespace {
+									return fmt.Errorf("path: spec.rules[%v]: a namespaced policy cannot clone resource in other namespace, expected: %v, received: %v", rule.Name, policyNamespace, rule.Generation.Clone.Namespace)
```

Comment:
Suggestion:

"a namespaced policy cannot clone resources to or from other namespaces"

Reply:
fixed